account-kit/rn-signer/ios/implementation/KeychainHelper.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Security
+import CryptoKit
+
+let account_kit_ephemeral_private_key = "ephemeralPrivateKey"
+
+// Keep save_ephemeral_private_key_query mutable
+var save_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecAttrAccessible as String: kSecAttrAccessibleAfterFirstUnlock
+]
+
+let get_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecReturnData as String: true,
+    kSecMatchLimit as String: kSecMatchLimitOne
+]
+
+let delete_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key
+]
+
+
+class KeychainHelper {
```

Comment:
Sorry I missed this before but any reason to not make this a struct with static functions instead of using a `shared`?

Reply:
There could be a good reason for it, but I just figured it'd be more swifty this way given there are not instance variables or mutated state inside the class itself?

Reply:
If it were me I'd opt to just have the a static struct as the helper like this:

```
struct KeychainHelper {
    @available(*, unavailable) private init() {}
    
    static func savePrivateKeyToKeychain(_ privateKey: P256.KeyAgreement.PrivateKey) {
        // ...
    }

    static func getPrivateKeyFromKeychain() -> P256.KeyAgreement.PrivateKey? {
        // ...
    }

    static func deletePrivateKeyFromKeychain() {
        // ...
    }
}
```